src/main.cpp

```
@@ -1,42 +1,43 @@
     auto *zigbee = new ZigbeeProvider(manager.mqtt_client(), // NOLINT
                                       &manager);             // NOLINT
     manager.register_provider(zigbee);
 
-    QFile file(style_path);
-    if (file.open(QFile::ReadOnly | QFile::Text)) {
-      QTextStream stream(&file);
-      app.setStyleSheet(stream.readAll());
-      file.close();
+    QQmlApplicationEngine engine;
+    engine.rootContext()->setContextProperty("manager", &manager);
+    const QUrl url(QStringLiteral("qrc:SmartHome/src/Main.qml"));
```

Clang (14.0.6)

[warning] variable 'url' is not initialized

(cppcoreguidelines-init-variables)